hvantk/commands/expression_atlas_downloader.py

```
@@ -8,12 +8,76 @@
+                           f"Retrying ({attempt}/{max_attempts}) in {delay} seconds...")
+            
+            # Wait before retrying with exponential backoff and jitter
+            time.sleep(delay + random.uniform(0, 1))
+            delay *= 2  # Exponential backoff
+            
+            # Re-establish FTP connection if URL is provided
+            if ftp_url:
+                try:
+                    ftp.quit()
+                except:
+                    pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)